### PR TITLE
Add documentation for logging addon

### PR DESCRIPTION
**Pull request by bashofmann**
only merge after deployment of addon

### CODE REVIEW

File: pages/03.addons/12.metakube-logging/default.en.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+taxonomy:
+    tag:
+        - metakube
+        - logging
+        - loki
+        - grafana
+        - addon
+---
+
+The log management Add-On installs a [Loki](https://github.com/grafana/loki/blob/master/docs/README.md) log management stack that integrates into the Grafana instace from the [Monitoring Add-On](../03.metakube-monitoring/default.en.md).
```
Comment (ekeih) [inline pages/03.addons/12.metakube-logging/default.en.md]:
```suggestion
The log management Add-On installs a [Loki](https://github.com/grafana/loki/blob/master/docs/README.md) log management stack that integrates into the Grafana instance from the [Monitoring Add-On](../03.metakube-monitoring/default.en.md).
```

File: pages/03.addons/12.metakube-logging/default.en.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Because of this the Monitoring Add-On is required to be installed before installing the Log Management Add-On.
+
+All logs provided via stdout or sterr by Pods in the cluster will be collected and stored for the configured retention time.
```
Comment (ekeih) [inline pages/03.addons/12.metakube-logging/default.en.md]:
```suggestion
All logs provided via stdout or stderr by Pods in the cluster will be collected and stored for the configured retention time.
```

File: pages/03.addons/12.metakube-logging/default.en.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+The following customization options are possible:
+
+### Loki configuration options
+
+| Option | Description | Default value |
+| ------ | ----------- | ------------- |
+| Retention Time | Amount of weeks the logs should be kept | 1 |
+| Persistent Storage | How large should the persistent data volume be | 10Gi |
+| Reject samples older | Log messages older then this value will never be stored | 168h |
```
Comment (ekeih) [inline pages/03.addons/12.metakube-logging/default.en.md]: Could this be misunderstood in a way that loki drops data after 168 hours?
Maybe something like "If Promtail sends messages older then this value they will be rejected by Loki"?

Comment (bashofmann) [reply pages/03.addons/12.metakube-logging/default.en.md]:
```suggestion
| Reject samples older | If Promtail sends messages older then this value they will be rejected by Loki | 168h |
```
Yes, your version is much better.

**Pull request merged**